### PR TITLE
build: support sepolia

### PR DESCRIPTION
**Pull request by TTNguyenDev**
## Dework Task
https://app.dework.xyz/nation3/development-guild-43499?taskId=28308d89-f718-467d-9306-3efc01daf7ed

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -123,12 +123,14 @@ https://github.com/nation3/foundations/blob/main/deployments/goerli.json
 
 https://github.com/nation3/foundations/blob/main/deployments/sepolia.json
 
-- `utils/PassportUtils.sol`: [`0x4Db31016fe58292B7654C9EDF8Ad106761BB7a90`](https://sepolia.etherscan.io/address/0x4Db31016fe58292B7654C9EDF8Ad106761BB7a90)
-  - > `⚠️ Not yet functional`
-- `citizens/DeveloperSkillLevels.sol`: [`0xcD441F74016bB50ff25959805F6466AdB49950D2`](https://sepolia.etherscan.io/address/0xcD441F74016bB50ff25959805F6466AdB49950D2)
-- `citizens/OperatorSkillLevels.sol`: [`0xb2731b1DB731Bc653ef3286264373BF1306B3e4B`](https://sepolia.etherscan.io/address/0xb2731b1DB731Bc653ef3286264373BF1306B3e4B)
-- `citizens/MarketeerSkillLevels.sol`: [`0x02FF5feb29dF1c03bD6fDd60C03Ac1375045Bd60`](https://sepolia.etherscan.io/address/0x02FF5feb29dF1c03bD6fDd60C03Ac1375045Bd60)
-- NationCred.sol: `0x...`
+- `utils/PassportUtils.sol`: [`0x4C72e8f37a2652BA6eEE956Ab30Ff21C3514cb5a`](https://sepolia.etherscan.io/address/0x4C72e8f37a2652BA6eEE956Ab30Ff21C3514cb5a)
```
Comment (aahna-ashina) [inline README.md:126]: Contract verification missing:  https://sepolia.etherscan.io/address/0x4C72e8f37a2652BA6eEE956Ab30Ff21C3514cb5a#code

File: README.md [hunk 2]
```diff
@@ -123,12 +123,14 @@ https://github.com/nation3/foundations/blob/main/deployments/goerli.json
...
+- `citizens/DeveloperSkillLevels.sol`: [`0xF7639A56A63a3373E235062273C463eF6203101D`](https://sepolia.etherscan.io/address/0xF7639A56A63a3373E235062273C463eF6203101D)
```
Comment (aahna-ashina) [inline README.md:127]: Verification missing:  https://sepolia.etherscan.io/address/0xF7639A56A63a3373E235062273C463eF6203101D#code

**Review by aahna-ashina — CHANGES_REQUESTED** [review]
Looks great!  Just the Etherscan verification missing 😅

File: README.md (hunk 1 above)
Comment (TTNguyenDev) [reply README.md:126]: Yeah, I just switched to WIP. My plan is to first deploy everything and then learn how to do contract verification later.

**Comment by TTNguyenDev** [comment]
@aahna-ashina Resolved

**Review by aahna-ashina — APPROVED** [review]

**Pull request merged**